pkgs/applications/networking/cluster/k8sgpt/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "k8sgpt";
+  version = "0.2.9";
+
+  src = fetchFromGitHub {
+    owner = "k8sgpt-ai";
+    repo = "k8sgpt";
+    rev = "v${version}";
+    hash = "sha256:039yiwyh84l1dm6b4vzabv427kfpxj41bjs53cqq85xmqah34bdm";
+  };
+
+  vendorHash = "sha256-nutpoc8UAPXgvBr4hUTYq9/d5OQXGOl3QNtaWhH9Cpw=";
+
+  CGO_ENABLED = 0;
+
+  ldflags = [
+    "-s" "-w"
+    "-X main.version=v${version}"
```

Comment:
```suggestion
    "-X main.version=v${version}"
    "-X main.commit=${src.rev}"
    "-X main.date=1970-01-01-00:00:01"
```

Reply:
@ivankovnatsky I'm not getting the date part. Is this how we specify the date while packaging in nix? Will it consider the time at which the package is being built?

Reply:
I think in some of my PRs Sandro suggest me to use this, I guess that's for compatibility purposes.